QATCH/common/licenseManager.py

```
@@ -0,0 +1,865 @@
+    """
+
+    _TRIAL_PERIOD = 90
+
+    def __init__(self,
+                 dbx_conn: dropbox.Dropbox,
+                 license_directory: str = "/device-licenses",
+                 auto_register_trial: bool = True,
+                 trial_duration_days: int = _TRIAL_PERIOD,
+                 cache_enabled: bool = True,
+                 cache_duration_hours: int = 24,
+                 cache_dir: str = None,
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)